integration/benchnet2/Makefile

```diff
@@ -4,6 +4,9 @@
 FLOW_GO_TAG = v0.28.15
 DOCKER_TAG := $(FLOW_GO_TAG)
 
+# default value of the Docker base registry URL which can be overriden when invoking the Makefile
+DOCKER_REGISTRY := gcr.io/flow-container-registry
```

Comment:
The Docker Registry should look like this, `us-west1-docker.pkg.dev/dl-flow-benchnet-automation/benchnet](http://us-west1-docker.pkg.dev/dl-flow-benchnet-automation/benchnet` because we are using artifact registry actually